packages/components/pages/f-account-info/src/components/AccountInfo.vue

```diff
@@ -81,94 +124,100 @@
             :class="[$style['c-accountInfo-changePasswordButton']]"
             data-test-id="account-info-submit-button"
             button-type="secondary"
+            href="/change-password?returnurl=/account/info"
             button-size="large"
             is-full-width
             action-type="submit">
             {{ $t('buttons.changePassword') }}
         </f-button>
 
-        <hr>
-
-        <p>{{ $t('deleteAccountMessage') }}</p>
-
-        <f-link
-            is-distinct
-            href="/help"
-            target="_blank">
-            {{ $t('deleteAccountLink') }}
-        </f-link>
-    </card>
+        <delete-account />
+    </f-card>
 </template>
 
 <script>
 import { VueGlobalisationMixin } from '@justeat/f-globalisation';
 
-import Card from '@justeat/f-card';
+import FErrorMessage from '@justeat/f-error-message';
+import '@justeat/f-error-message/dist/f-error-message.css';
+
+import FCard from '@justeat/f-card';
 import '@justeat/f-card/dist/f-card.css';
 
 import FormField from '@justeat/f-form-field';
 import '@justeat/f-form-field/dist/f-form-field.css';
 
-import FLink from '@justeat/f-link';
-import '@justeat/f-link/dist/f-link.css';
-
 import FButton from '@justeat/f-button';
 import '@justeat/f-button/dist/f-button.css';
 
+import '@justeat/f-link/dist/f-link.css';
+
+import EmailAddressField from './EmailAddressField.vue';
+import DeleteAccount from './DeleteAccount.vue';
+
+import AccountInfoValidationsMixin from './AccountInfoValidationMixin.vue';
 import tenantConfigs from '../tenants';
 
 export default {
     components: {
-        Card,
+        FCard,
         FormField,
-        FLink,
-        FButton
+        FButton,
+        FErrorMessage,
+        EmailAddressField,
+        DeleteAccount
     },
 
     mixins: [
-        VueGlobalisationMixin
+        VueGlobalisationMixin,
+        AccountInfoValidationsMixin
     ],
 
     props: {
         locale: {
             type: String,
-            default: ''
+            default: 'en-GB'
```

Comment:
I think `VueGlobalisationMixin` already exposes the `locale` property.

Reply:
I should know that really given I wrote it :D